satellite/marketing/marketingweb/server.go

```diff
@@ -128,20 +170,73 @@ func (s *Server) parseTemplates() (err error) {
 	return nil
 }
 
+// timeConverter formats form time input as time.Time.
+func timeConverter(value string) reflect.Value {
+	v, err := time.Parse("2006-01-02", value)
+	if err != nil {
+		return reflect.Value{}
```

Comment:
Add a note that this is the invalid value for decoder.

Reply:
I think @egonelbre meant to add a comment here? I don't think we need to log that